app/api/attendees.py

```diff
@@ -56,8 +56,9 @@ def before_post(self, args, kwargs, data):
                 "Ticket belongs to a different Event"
             )
         # Check if the ticket is already sold out or not.
-        if get_count(db.session.query(TicketHolder.id).
-                     filter_by(ticket_id=int(data['ticket']), deleted_at=None)) >= ticket.quantity:
+        if get_count(db.session.query(TicketHolder).
```

Comment:
you are referring to `TicketHolder.user` if I am buying a ticket for 3 others which ae not in the system then ticketHolde.user will be none for them.

Reply:
Currently, how do we go about it? When we want to book tickets for other users, do they need to be registered?

Reply:
Okay so they don't have to be registered, checked locally. So the point is valid.